.travis.yml

```diff
@@ -36,12 +38,13 @@ before_script:
   - if [ "$SYMFONY_VERSION" != "" ]; then composer require --no-update "symfony/symfony:${SYMFONY_VERSION}"; fi;
   - if [ "$DEPENDENCIES" != "low" ]; then composer update; fi;
   - if [ "$DEPENDENCIES" = "low" ]; then composer update --prefer-lowest; fi;
+  - if [ "$SKIP_INTEROP" != "" ]; then composer remove --dev container-interop/container-interop; fi;
```

Comment:
this should be before the `composer update` commands (with a `composer remove --no-update`. Otherwise, we might loose the `--prefer-lowest` effect.

Reply:
Thank you!